OOPAssignment1/Program.cs

```diff
@@ -11,29 +11,15 @@ class Program
     {
         static void Main()
         {
-            //Local list of integers to hold the first five measurements of the text
-            List<int> parameters = new List<int>();
+            Input input = new Input();
+            input.FileTextInput("example.txt");
 
```

Comment:
A choice between manual text or pulling from a text file might need to be included, as it only pulls from the example.txt thus far.